aws/main.tf

```diff
@@ -217,7 +217,8 @@ module "ec2_database" {
 }
 
 module "ec2_asg_indexer" {
-  source = "./asg"
+  source        = "./asg"
+  block_devices = var.block_devices
```

Comment:
why does this need to be added to each asg?

Reply:
- It's already tested for staging.
- it's for all because I modifies the terraform scripts to be more customizable now, previously the block device was hard coded and the same would be used for staging and test but now with this change we can set different block device configuration for stage and test.